features/profile_page.feature

```diff
@@ -8,6 +8,7 @@ Feature: profile page and edit page
     Given the following users exist:
       | first_name | last_name | major | gender | grad_year | ethnicity  | transfer | graduate | international |
       | Tony       | Li        | IEOR  | Male   | 2020      | East Asian | false    | false    | false         |
+    And I am logged in
```

Comment:
Just curious if we're still handling the logged in through the interface or the rack session middleware you added? Either way both would work, I assume!

Reply:
The rack session middleware overwrites this, I think.